pkg/module/all_test.go

```diff
@@ -10,8 +10,8 @@ import (
 
 const (
 	// these values need to point to a real repository that has a tag
-	repoURI = "github.com/arschles/assert"
```

Comment:
intentional?

Reply:
yes, i've changed it to NYTimes/gizmo because it's an example of a module name which needs to be encoded (!n!y!times) and I wanted to cover that as well.